### DISCUSSION TITLE
How to put away or show the sidebar?

### DISCUSSION THREAD

**Discussion by Sunny19191** [web-infra-dev/rspress#1132] [Ideas]
Offer a plugin to support the sidebar put away and show，maybe add a button to realization， which like the function of scroll to top

**Comment by Timeless0911** [comment]
The sidebar is a responsive design, when the screen width is not enough, it is the same as what you described.

More custom behaviors can be implemented by yourself through plugins, overriding styles, custom themes, etc.

**Reply by Sunny19191** [reply]
All right. Aim to display the table's column not wrap the text by aotomatic. So we want to set the sidebar put away and the center content is able to display enough. Do you have any idea about it?

**Reply by Timeless0911** [reply]
You can modify table style like `table-layout` property, see MDN for help.
